src/tests/fake/indexes.ts

```diff
@@ -0,0 +1,78 @@
+import { faker } from "@faker-js/faker";
+import { merge } from "lodash";
+import nock from "nock";
+import { IndexMinimal, IndexNested } from "../../js/indexes/types";
+import { JobMinimal } from "../../js/jobs/types";
+import { ReferenceNested } from "../../js/references/types";
+import { UserNested } from "../../js/users/types";
+import { createFakeJobMinimal } from "./jobs";
+import { createFakeReferenceNested } from "./references";
+import { createFakeUserNested } from "./user";
+import { FakeSearchResults } from "./utils";
+
+type CreateFakeIndexNestedProps = {
+    id?: string;
+    version?: number;
+};
+
+export function createFakeIndexNested(props?: CreateFakeIndexNestedProps): IndexNested {
+    const defaultIndexNested = {
+        id: faker.random.alphaNumeric(8),
+        version: faker.datatype.number(5),
+    };
+
+    return merge(defaultIndexNested, props);
+}
+
+type createFakeIndexMinimalProps = CreateFakeIndexNestedProps & {
```

Comment:
I believe this is TS convention. This inconsistency is introduced in a few other spots as well.
```suggestion
type CreateFakeIndexMinimalProps = CreateFakeIndexNestedProps & {
```